src/WebJobs.Script.WebHost/Management/LinuxSpecialization/BashCommandHandler.cs

```diff
@@ -41,7 +40,7 @@ public BashCommandHandler(IMetricsLogger metricsLogger, ILogger<BashCommandHandl
                             CreateNoWindow = true
                         }
                     };
-                    _logger.LogInformation($"Running: {process.StartInfo.FileName} {process.StartInfo.Arguments}");
+                    _logger.LogInformation($"Running: {process.StartInfo.FileName}");
```

Comment:
Recommend either removing this line completely at this point, or updating it to make it clear that arguments are omitted, e.g. "Running: bash.exe (arguments omitted)"

Reply:
Thanks. I'll go with "Running: bash.exe (arguments omitted) approach. 

I noticed the logging comes after this section, so that these logs became inconsistent. Also this logging is used for Integration testing. So that keep the Running logs with your recommendation will be great.